components/shopify_partner/shopify_partner.app.mjs

```diff
@@ -22,9 +22,15 @@ export default {
       type: "string",
       description:
         "Only include events up to this specific time (ISO timestamp)",
-      label: "occurredAtMin",
+      label: "occurredAtMax",
       optional: true,
     },
+    paginationEnabled: {
+      type: "boolean",
+      label: "Paginate results",
+      description: "Paginate through all of your Shopify Partner records until occurredAtMin is reached. This can cause many invocations.",
```

Comment:
`occurredAtMax`?

Reply:
"This can cause many invocations" -> "This can cause many invocations **in linked workflows**" correct? i.e. the source should still only run once, paginate, and emit right?